Gemfile

```diff
@@ -22,6 +22,12 @@ if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.2.2")
   gem 'rack', '< 2.0'
 end
 
+if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.2.2")
```

Comment:
can we roll this into the conditional above? and maybe a comment for the reason why it's there